WordPress/gutenberg
# Support custom units on theme.json

Related #20588
Alternative to #24873
Similar to #24761 but for custom units.

The idea of this PR is to support configuring custom unit support from theme.json.

**Testing instructions**

* Check that you can config custom units using the theme.json `spacing.units` path.
* Check that you can still use the `custom-units` theme support flag.
* Check that it's opt-out now.
